### PR TITLE
feat: quick install scripts

### LINKED ISSUE #43: [Enhancement]: Automate installation steps for verify and double-check documentation
### Contact Details

emanuel@ongres.com

### What this feature/enhancement tries to solve?

There is an initial work for testing the installation steps over a variety of Linux-flavors under Docker env:

https://github.com/ongres/babelfish_install_tests

### If want to provide us a more details about how to implement.

_No response_

### Version

babelfishpg_tsql (Default)

### Relevant documentation

_No response_

### Code of Conduct

- [X] I agree to follow this project's Code of Conduct

### PR DESCRIPTION
**Pull request by xaviersierra**
### Description

Provides scripts for quick installing babelfish
 
### Check List
- [x] Commits are signed per the DCO using --signoff 

By submitting this pull request, I confirm that my contribution is under the terms of the Apache 2.0 and PostgreSQL licenses, and grant any person obtaining a copy of the contribution permission to relicense all or a portion of my contribution to the PostgreSQL License solely to contribute all or a portion of my contribution to the PostgreSQL open source project.

For more information on following Developer Certificate of Origin and signing off your commits, please check [here](https://github.com/babelfish-for-postgresql/babelfish_extensions/blob/main/CONTRIBUTING.md#developer-certificate-of-origin).

### CODE REVIEW

**Review by nasbyj — CHANGES_REQUESTED** [review]
Haven't done a full review yet, but please add an appropriate reference to the install instructions to `/README.md`

**Review by 3manuek — DISMISSED** [review]

File: install/README.md [hunk 1]
```diff
@@ -0,0 +1,23 @@
...
+
+- Centos 7
+- Centos 8
+- Ubuntu 20.04
+- Debian bullseye
+- Fedora 36
+- Amazon Linux 2
+
+## Install instructions configurations
+The are some environment variable that can be set in order to customize the installation script.
```
Comment (susanmdouglas-aws) [inline install/README.md]: How about:

You can use environment variables to customize the installation script:

File: install/README.md [hunk 2]
```diff
@@ -0,0 +1,23 @@
+# Quick install scripts
+
+This directory contains scripts for installing Babelfish for PostgreSQL. 
+
+By running the install.sh script it will perform the installation process for your linux distribution. 
+
+Currently the supported linux distribution for this script are:
```
Comment (susanmdouglas-aws) [inline install/README.md]: How about:

This directory contains scripts that install Babelfish for PostgreSQL.  Select the install.sh script for your Linux platform to install Babelfish on your system. 

Currently the supported Linux distributions are:

File: install/README.md [hunk 3]
```diff
@@ -0,0 +1,23 @@
...
+
+- Centos 7
+- Centos 8
+- Ubuntu 20.04
+- Debian bullseye
```
Comment (susanmdouglas-aws) [inline install/README.md:10]: The Debian website capitalizes Bullseye...

**Comment by 3manuek** [comment]
WIP tasks:

- Adding docker environments to serve as a test environment for the installation scripts.
- Documenting the test procedure.

cc/ @gopinathpai @nasbyj

**Pull request closed without merging**